guides/release/upgrading/editions.md

```diff
@@ -0,0 +1,1078 @@
+  value to an argument, it must be wrapped in double curlies:
+
+  ```hbs
+  <Todo @done={{false}}/>
+  ```
+
+  This is necessary because, like HTML, all values that are passed to attributes
+  and arguments which are not wrapped as strings are coerced into strings.
+
+  In classic components, arguments passed this way will still be assigned to the
+  instance just like normal. The behavior of arguments in Glimmer components
+  will be discussed later one.
+
+- Yielded values work the same as in curly invocation:
+
+  ```hbs
+  <TodoList as |item|>
+    <Todo @item={{item}}/>
+  </TodoList>
+  ```
+
+- Yielded components can also be invoked with angle bracket syntax:
+
+  ```hbs
+  <TodoList as |Item|>
+    <Item/>
+  </TodoList>
+  ```
+
+- Positional arguments are _not_ available in angle bracket invocation, since
+  there is some abiguity between their behavior and the behavior of standard
+  HTML attributes (HTML attributes without `=` default to truthy). If you still
+  need positional arguments, you _must_ use the component with curly bracket
+  syntax.
+
+Angle bracket syntax works with both classic components and Glimmer components,
+so you can update your templates incrementally, independently from updating the
+components themselves. It is also not a _requirement_, curly invocation
+continues to work with classic components and Glimmer components, but it is
+highly recommended in general.
+
+#### Named Arguments
+
+Named arguments are a new syntax that allows you to refer to the arguments of a
+component _directly_ within your template:
+
+```hbs {data-filename=application.hbs}
+<BlogPost @title="Hello, world!"/>
+```
+
+```hbs {data-filename=blog-post.hbs}
+<h1>{{@title}}</h1>
+```
+
+As you can see, named arguments are prefixed with the `@` symbol, which mirrors
+the argument being passed into a component via angle bracket invocation, making
+it easier to remember and connect the two! However, you can still refer to
+argument values if the component was invoked using curly bracket syntax:
+
+```hbs {data-filename=application.hbs}
+{{blog-post title="Hello, world!"}}
+```
+
+```hbs {data-filename=blog-post.hbs}
+<!-- This still works -->
+<h1>{{@title}}</h1>
+```
+
+This means you can safely refactor your components to use named argument syntax
+without having to worry about every single _use_ of your component being updated
+to angle bracket syntax.
+
+Another very important thing to know about named argument syntax is that it
+_always_ refers to the _original_ value of the argument. If you change that
+value in a classic component, it will _not_ update:
+
+```js {data-filename=blog-post.js}
+export default Component.extend({
+  init() {
+    this.set('title', this.title.toUpperCase());
+  },
+});
+```
+
+```hbs {data-filename=blog-post.hbs}
+<!-- This is the original title, "Hello, world!" -->
+<h1>{{this.title}}</h1>
+
+<!-- This is the uppercased title, "HELLO, WORLD!" -->
+<h1>{{@title}}</h1>
```

Comment:
Should these be reversed?

Reply:
Yup, thanks for pointing that out!

Reply:
No problem. I definitely agree with @ddoria921, really good work. I think this reads clearly and concisely, despite having to cover a *lot* of ground. Well done!